pyomo/util/latex_printer.py

```diff
@@ -0,0 +1,619 @@
+        precedence = node.PRECEDENCE
+    else:
+        # Should never hit this
+        precedence = -1
+
+    if childPrecedence[0] > precedence:
+        arg1 = ' \\left( ' + arg1 + ' \\right) '
+
+    if childPrecedence[1] > precedence:
+        arg2 = ' \\left( ' + arg2 + ' \\right) '
+
+    return "%s^{%s}" % (arg1, arg2)
+
+
+def handle_abs_node(node, arg1):
+    return ' \\left| ' + arg1 + ' \\right| '
+
+
+def handle_unary_node(node, arg1):
+    fcn_handle = node.getname()
+    if fcn_handle == 'log10':
+        fcn_handle = 'log_{10}'
+
+    if fcn_handle == 'sqrt':
+        return '\\sqrt { ' + arg1 + ' }'
+    else:
+        return '\\' + fcn_handle + ' \\left( ' + arg1 + ' \\right) '
+
+
+def handle_equality_node(node, arg1, arg2):
+    return arg1 + ' = ' + arg2
+
+
+def handle_inequality_node(node, arg1, arg2):
+    return arg1 + ' \leq ' + arg2
+
+
+def handle_var_node(node):
+    name = node.name
+
+    splitName = name.split('_')
+
+    filteredName = []
+
+    prfx = ''
+    psfx = ''
+    for i in range(0, len(splitName)):
+        se = splitName[i]
+        if se != 0:
+            if se == 'dot':
+                prfx = '\dot{'
+                psfx = '}'
+            elif se == 'hat':
+                prfx = '\hat{'
+                psfx = '}'
+            elif se == 'bar':
+                prfx = '\\bar{'
+                psfx = '}'
+            else:
+                filteredName.append(se)
+        else:
+            filteredName.append(se)
+
+    joinedName = prfx + filteredName[0] + psfx
+    for i in range(1, len(filteredName)):
+        joinedName += '_{' + filteredName[i]
+
+    joinedName += '}' * (len(filteredName) - 1)
+
+    return joinedName
+
+
+def handle_num_node(node):
+    if isinstance(node, float):
+        if node.is_integer():
+            node = int(node)
+    return str(node)
+
+
+def handle_sum_expression(node, *args):
+    rstr = args[0]
+    for i in range(1, len(args)):
+        if args[i][0] == '-':
+            rstr += ' - ' + args[i][1:]
+        else:
+            rstr += ' + ' + args[i]
+    return rstr
+
+
+def handle_monomialTermExpression_node(node, arg1, arg2):
+    if arg1 == '1':
+        return arg2
+    elif arg1 == '-1':
+        return '-' + arg2
+    else:
+        return arg1 + arg2
+
+
+def handle_named_expression_node(node, arg1):
+    return arg1
+
+
+def handle_ranged_inequality_node(node, arg1, arg2, arg3):
+    return arg1 + ' \\leq ' + arg2 + ' \\leq ' + arg3
+
+
+def handle_exprif_node(node, arg1, arg2, arg3):
+    return 'f_{\\text{exprIf}}(' + arg1 + ',' + arg2 + ',' + arg3 + ')'
+
+    ## Raises not implemented error
+    # raise NotImplementedError('Expr_if objects not supported by the Latex Printer')
+
+    ## Puts cases in a bracketed matrix
+    # pstr = ''
+    # pstr += '\\begin{Bmatrix} '
+    # pstr += arg2 + ' , & ' + arg1 + '\\\\ '
+    # pstr += arg3 + ' , & \\text{otherwise}' + '\\\\ '
+    # pstr += '\\end{Bmatrix}'
+    # return pstr
```

Comment:
Remove?

Reply:
No I want to keep this because of the ambiguity of how to handle this.  Someone in the future may wish to look at this

Reply:
You might include a comment that the way to handle this in the future might be to define a function of the inputs using `cases` and then substitute that function in place of the exprIf wherever it appears.